aiogram/types/message.py

```diff
@@ -193,7 +193,7 @@ def parse_entities(self, as_html=True):
 
         quote_fn = md.quote_html if as_html else md.escape_md
 
-        if not self.entities:
+        if not (self.entities or self.caption_entities):
```

Comment:
You can move that to variable (`entities = ...`) and then use it in other places